contrib/fuzz/go.mod

```diff
@@ -0,0 +1,143 @@
+module github.com/containerd/containerd/contrib/fuzz
+
+replace github.com/containerd/containerd => ../../
+
+replace github.com/containerd/containerd/cmd => ../../cmd
+
+go 1.18
+
+require (
+	github.com/AdaLogics/go-fuzz-headers v0.0.0-20220903154154-e8044f6e4c72
+	github.com/containerd/containerd v1.6.1
```

Comment:
Note that whoever would try to `go install github.com/containerd/containerd/cmd/containerd@version` will get this as dependency, _NOT_ the latest version of the containerd code.

So whenever a release is done, we must (again)

- do a release of the main module (`github.com/containerd/containerd`) (release of the containerd library)
- update the `cmd` module to have the new release
- tag/release the `cmd` module (binary release)

Reply:
Ah, whoops, meant to comment on the other `go.mod` (same applies though)